docs/source/user/installing.rst

```diff
@@ -185,18 +185,28 @@ vary with target operating system and hardware. The following lists them all
   * ``numpy``
   * ``llvmlite``
   * Compiler toolchain mentioned above
-  * OpenMP C headers and runtime libraries compatible with the compiler
-    toolchain mentioned above and accessible to the compiler via standard flags
-    (Linux, Windows).
 
-* Optional build time:
-
-  * ``llvm-openmp`` (OSX) - provides headers for compiling OpenMP support into
+* Build time toggles:
+  * ``NUMBA_NO_OPENMP=1``
+    DEFAULT: undefined
+    set the environment variable to ``1`` when building using ``setup.py``
+    to disable compilations of OpenMP threading backend.
+    If enabled you need OpenMP C headers and runtime libraries compatible
+    with the compiler toolchain mentioned above and accessible to the
+    compiler via standard flags (Linux, Windows)
+    * ``llvm-openmp`` (OSX) - provides headers for compiling OpenMP support into
     Numba's threading backend
-  * ``intel-openmp`` (OSX) - provides OpenMP library support for Numba's
+    * ``intel-openmp`` (OSX) - provides OpenMP library support for Numba's
     threading backend.
-  * ``tbb-devel`` - provides TBB headers/libraries for compiling TBB support
-    into Numba's threading backend
+  * ``NUMBA_NO_TBB=1 TBBROOT=/usr``
+    DEFAULT: both undefined
+    set the environment variable ``NUMBA_NO_TBB`` to ``1`` when building using
+    ``setup.py`` to disable the compilation of the TBB threading backend.
+    If not setting it to ``1`` you can also add the location where the
+    TBB headers are going to be searched for, which is generally in ``/usr``,
+    as they are installed in ``/usr/include/tbb/tbb.h``.
```

Comment:
This is very linux specific and also assumes it's a system package manager installed package. I think it might be appropriate to just mention that `TBBROOT` can be set if needed and to just link to the Intel documentation on it, https://software.intel.com/content/www/us/en/develop/documentation/advisor-user-guide/top/appendix/adding-parallelism-to-your-program/adding-the-parallel-framework-to-your-build-environment/defining-the-tbbroot-environment-variable.html ? What do you think?

Reply:
This is the location in gentoo, arch and ubuntu, afaik. So writing it out explicitly is a good thing, in addition to linking to the intel documentation.

Will add the intel link.

Reply:
It may well be the location on a few linux distros, but what about windows and OSX users and people who aren't using linux installation defaults? I think the OS invariant link is more suitable.

Reply:
OK, I've added both the things for now, would that be acceptable?